staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -2446,6 +2446,25 @@ type Container struct {
 	// +optional
 	// +listType=atomic
 	ResizePolicy []ContainerResizePolicy `json:"resizePolicy,omitempty" protobuf:"bytes,23,rep,name=resizePolicy"`
+	// Restart policy for the container.
+	// If value is not set (default), the restart behavior is defined by
+	// the Pod's RestartPolicy.
```

Comment:
I don’t think the new format is more understandable - especially the list of clauses below.  This is documentation for humans, and we should use the first few sentences to explain in plain English what the purpose of the field is.

Reply:
rephrased them.